RecoTracker/PixelSeeding/plugins/alpaka/CAHitNtupletGenerator.cc

```diff
@@ -36,16 +37,37 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
 
     //Common Params
     void fillDescriptionsCommon(edm::ParameterSetDescription& desc) {
-      // 87 cm/GeV = 1/(3.8T * 0.3)
-      // take less than radius given by the hardPtCut and reject everything below
-      // auto hardCurvCut = 1.f/(0.35 * 87.f);
+      desc.add<double>("cellZ0Cut", 12.0f)->setComment("Z0 cut for cells");
+      desc.add<double>("cellPtCut", 0.5f)->setComment("Preliminary pT cut at cell building level.");
+
+      //// Pixel Cluster Cuts (@cell level)
+      desc.add<double>("dzdrFact", 8.0f * 0.0285f / 0.015f);
+      desc.add<unsigned int>("minYsizeB1", 1)
+          ->setComment("Cut on inner hit cluster size (in Y) for barrel-forward cells. Barrel 1 cut.");
+      desc.add<unsigned int>("minYsizeB2", 1)
+          ->setComment("Cut on inner hit cluster size (in Y) for barrel-forward cells. Barrel 2 cut.");
+      desc.add<unsigned int>("maxDYsize12", 28)
+          ->setComment("Cut on cluster size differences (in Y) for barrel-forward cells. Barrel 1-2 cells.");
+      desc.add<unsigned int>("maxDYsize", 20)
+          ->setComment("Cut on cluster size differences (in Y) for barrel-forward cells. Other barrel cells.");
+      desc.add<unsigned int>("maxDYPred", 20)
+          ->setComment("Cut on cluster size differences (in Y) for barrel-forward cells. Barrel-forward cells.");
+
+      // Container sizes
+      desc.add<std::string>("maxNumberOfDoublets", std::to_string(pixelTopology::Phase1::maxNumberOfDoublets));
+      desc.add<std::string>("maxNumberOfTuples", std::to_string(pixelTopology::Phase1::maxNumberOfTuples));
+      desc.add<double>("avgHitsPerTrack", 5.0f);
+      desc.add<double>("avgCellsPerHit", 25.0f);
+      desc.add<double>("avgCellsPerCell", 2.0f);
+      desc.add<double>("avgTracksPerCell", 1.0f);
+
+      // nTuplet Cuts and Params
       desc.add<double>("ptmin", 0.9f)->setComment("Cut on minimum pt");
-      desc.add<double>("CAThetaCutBarrel", 0.002f)->setComment("Cut on RZ alignement for Barrel");
-      desc.add<double>("CAThetaCutForward", 0.003f)->setComment("Cut on RZ alignment for Forward");
+      //// 87 cm/GeV = 1/(3.8T * 0.3)
+      //// take less than radius given by the hardPtCut and reject everything below
       desc.add<double>("hardCurvCut", 1.f / (0.35 * 87.f))
```

Comment:
what are `0.3` in the comment above and `0.35` in the formula here ?

Reply:
The `0.35` is the min pt in GeV corresponding to the  `hardCurvCut`. The `0.3` comes from the conversion to natural units to estimate the radius of a 1GeV track.